Testing config requires `openzwavemqtt`.

```
____________ ERROR collecting tests/components/config/test_zwave.py ____________
ImportError while importing test module '/build/source/tests/components/config/test_zwave.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/wkw6fsjasr7jbbrlakxxpbiapa8hws42-python3-3.8.7/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/components/config/test_zwave.py:9: in <module>
    from homeassistant.components.zwave import DATA_NETWORK, const
homeassistant/components/zwave/__init__.py:40: in <module>
    from . import const, websocket_api as wsapi, workaround
homeassistant/components/zwave/websocket_api.py:5: in <module>
    from homeassistant.components.ozw.const import DOMAIN as OZW_DOMAIN
homeassistant/components/ozw/__init__.py:6: in <module>
    from openzwavemqtt import OZWManager, OZWOptions
E   ModuleNotFoundError: No module named 'openzwavemqtt'
```

--- a/pkgs/development/python-modules/plexauth/default.nix
+++ b/pkgs/development/python-modules/plexauth/default.nix
@@ -2,14 +2,14 @@
 
 buildPythonPackage rec {
   pname = "plexauth";
-  version = "0.0.5";
+  version = "0.0.6";
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "jjlawren";
     repo = "python-plexauth";
     rev = "v${version}";
-    sha256 = "1wbrn22iywl4ccz64r3w3f17k0r7vi2cqkqd2mrdkx5xqhscn9hz";
+    sha256 = "1wy6syz3cgfw28zvigh2br4jabg8rzpi5l0bhfb6vjjc7aam13ag";
   };
 
   propagatedBuildInputs = [ aiohttp ];

--- a/pkgs/development/python-modules/python-tado/default.nix
+++ b/pkgs/development/python-modules/python-tado/default.nix
@@ -2,7 +2,7 @@
 
 buildPythonPackage rec {
   pname = "python-tado";
-  version = "0.9.0";
+  version = "0.11.0";
 
   disabled = pythonOlder "3.5";
 
@@ -10,7 +10,7 @@ buildPythonPackage rec {
     owner = "wmalgadey";
     repo = "PyTado";
     rev = version;
-    sha256 = "0cr5sxdvjgdrrlhl32rs8pwyay8liyi6prm37y66dn00b41cb5l3";
+    sha256 = "0fw4f9gqnhxwpxyb34qi8bl5pmzz13h4x3mdk903hhjyccanqncr";
   };
 
   propagatedBuildInputs = [ requests ];

--- a/pkgs/development/python-modules/volkszaehler/default.nix
+++ b/pkgs/development/python-modules/volkszaehler/default.nix
@@ -7,11 +7,11 @@
 
 buildPythonPackage rec {
   pname = "volkszaehler";
-  version = "0.2.0";
+  version = "0.2.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "087gw1k3f81lm859r0j65cjia8c2dcy4cx8c7s3mb5msb1csdh0x";
+    sha256 = "13zhxq08wn5y9yn6xbazfl0gxxysmirwpc26wcnr6jk2va1kpc4l";
   };
 
   propagatedBuildInputs = [

